.github/workflows/update-images.yaml

```diff
@@ -172,41 +170,54 @@ jobs:
 
     - name: Deploy all tasks
       run: |
-        export PATH=$PATH:$PWD/bin
+        source samples/kctf/activate
         cd samples
-        for challenge_name in $(kctf-kubectl get challenges -o "jsonpath={.items[*].metadata.name}"); do
-          kctf-kubectl delete "challenge/${challenge_name}"
+        for challenge_name in $(kubectl get challenges -o "jsonpath={.items[*].metadata.name}"); do
+          kubectl delete "challenge/${challenge_name}"
         done
         for f in *; do
-          if [ ! "$f" == "kctf-conf" ]; then
-            pushd $f
-              CHALLENGE_NAME="$(yq eval '.metadata.name' challenge.yaml)"
-              make start
-              # We want to wait for the deployment to be available, but it
-              # might not have been created yet by the operator.
-              # Ideally, we would expose the condition in the operator but I
-              # don't think that's currently possible.
-              for i in {1..5}; do
-                kctf-kubectl get "deployment/${CHALLENGE_NAME}" && break
-                echo "deployment/${CHALLENGE_NAME} doesn't exist yet, sleeping"
-                sleep 5
-              done
-              kctf-kubectl wait --for=condition=available --timeout=5m "deployment/${CHALLENGE_NAME}"
-              make stop
-            popd
+          if [ "$f" == "kctf" ]; then
+            continue
           fi
+          pushd $f
+            CHALLENGE_NAME="$("${KCTF_BIN}/yq" eval '.metadata.name' challenge.yaml)"
+            kctf chal start
+            # We want to wait for the deployment to be available, but it
+            # might not have been created yet by the operator.
+            # Ideally, we would expose the condition in the operator but I
+            # don't think that's currently possible.
+            for i in {1..5}; do
+              kubectl get "deployment/${CHALLENGE_NAME}" && break
+              echo "deployment/${CHALLENGE_NAME} doesn't exist yet, sleeping"
+              sleep 5
+            done
+            kubectl wait --for=condition=available --timeout=5m "deployment/${CHALLENGE_NAME}"
```

Comment:
whats the difference between this and the for loop above? I assume the answer is that `kubectl wait` fails if the deployment doesnt exist, but it would be nice to add a comment if thats the expectation here

Reply:
hm? Isn't that pretty much what the comment already says?:

```
# We want to wait for the deployment to be available, but it
# might not have been created yet by the operator.
```

Reply:
I mean, that this code is relying on the fact that kubectl wait will fail if the deployment hasn't been created after 5 times 5 seconds.

Reply:
Hmm, I thought I replied to this already. I added some text to the comment.